packages/react-storage/src/components/StorageBrowser/__tests__/createStorageBrowser.spec.tsx

```diff
@@ -32,6 +32,16 @@ const config = {
 const input = { config };
 
 describe('createStorageBrowser', () => {
+  beforeAll(() => {
+    // defining `crypto` here to allow `useDataState` to continue working as this test file
+    // is covering a fair amount of component code as a side effect. Not ideal and should be
+    // readdressed
+    let id = 0;
+    Object.defineProperty(globalThis, 'crypto', {
+      value: { randomUUID: () => ++id },
+    });
+  });
```

Comment:
Preference is to improve the testing of the code that is currently being tested as a side effect and not use this workaround but trying not to add additional scope to this PR